d2bs/kolbot/libs/bots/TeamDiablo.js

```diff
@@ -0,0 +1,462 @@
+            },
+            init: function () {
+                if (diablo.data.initialized) {
+                    return true;
+                }
+
+                var id, layout;
+
+                for (id in diablo.data.seals) {
+                    switch (diablo.util.getLayout(diablo.data.seals[id].layoutSeal, diablo.data.seals[id].layoutCoord)) {
+                        case 1:
+                            print('Layout A for ' + id);
+                            diablo.data.seals[id].path = diablo.data.paths[id].A;
+                            diablo.data.seals[id].spot = diablo.data.paths[id].ASpot;
+                            break;
+                        case 2:
+                            print('Layout B for ' + id);
+                            diablo.data.seals[id].path = diablo.data.paths[id].B;
+                            diablo.data.seals[id].spot = diablo.data.paths[id].BSpot;
+                            break;
+                        default:
+                            throw new Error('No such path available for ' + id + '. Layout is: ' + layout);
+                    }
+                }
+                diablo.data.initialized = true;
+                return true;
+            },
+            toChaos: function () {
+                if (me.area !== Areas.ChaosSanctuary) {
+                    Town.goToTown(4);
+                    // Wait 30 seconds for a tp
+                    if (!Team.util.takePortal(Areas.ChaosSanctuary, 30)) {
+                        // Failed to get to chaos
+                        return false;
+                    }
+                }
+                return this.init();
+            },
+            sort: function (a, b) {
+                if (Config.BossPriority) {
+                    if ((a.spectype & 0x5) && (b.spectype & 0x5)) {
+                        return getDistance(me, a) - getDistance(me, b);
+                    }
+
+                    if (a.spectype & 0x5) {
+                        return -1;
+                    }
+
+                    if (b.spectype & 0x5) {
+                        return 1;
+                    }
+                }
+
+                // Entrance to Star / De Seis
+                if (me.y > 5325 || me.y < 5260) {
+                    if (a.y > b.y) {
+                        return -1;
+                    }
+
+                    return 1;
+                }
+
+                // Vizier
+                if (me.x < 7765) {
+                    if (a.x > b.x) {
+                        return -1;
+                    }
+
+                    return 1;
+                }
+
+                // Infector
+                if (me.x > 7825) {
+                    if (!checkCollision(me, a, 0x1) && a.x < b.x) {
+                        return -1;
+                    }
+
+                    return 1;
+                }
+
+                return getDistance(me, a) - getDistance(me, b);
+            },
+            openSeal: function (classid) {
```

Comment:
You probably want to change the logic of openSeal to what is used in FastDiablo.js